aiida_quantumespresso/calculations/base.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+"""Defines a CalcJob base class for aiida-quantumespresso.
+
+The custom CalcJob base class automatically sets the'invalidates_cache'
+attribute of exit codes based on the status integer.
```

Comment:
Maybe add also that all `CalcJob` implementation _have to_ subclass from this class and not `aiida.engine`.